### PR TITLE
Composer packages updated

### PR DESCRIPTION
**Pull request by escopecz**
[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | N
| Automated tests included? | N
| Related user documentation PR URL | N
| Related developer documentation PR URL | N
| Issues addressed (#s or URLs) | N
| BC breaks? | N
| Deprecations? | N

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:

Upgrade of dependencies is a good thing to do from time to time. New versions contain bug fixes, security fixes, speed improvements and other enhancement.

This PR adds a new restriction in composer.json which will force Composer to use only packages for our lower PHP version. From now on it won't happen that it would update to some PHP 7-only packages.

Updated Composer dependencies with `composer update`. Result:
```
$ composer update
Loading composer repositories with package information
Updating dependencies (including require-dev)
Package operations: 1 install, 82 updates, 1 removal
  - Removing stecman/symfony-console-completion (0.7.0)
  - Updating symfony/debug (v2.8.28 => v2.8.34)
  - Updating symfony/property-access (v2.8.28 => v2.8.34)
  - Updating symfony/polyfill-mbstring (v1.6.0 => v1.7.0)
  - Updating symfony/options-resolver (v2.8.28 => v2.8.34)
  - Updating symfony/polyfill-php54 (v1.6.0 => v1.7.0)
  - Updating symfony/polyfill-intl-icu (v1.6.0 => v1.7.0)
  - Updating symfony/intl (v2.8.28 => v2.8.34)
  - Updating symfony/event-dispatcher (v2.8.28 => v2.8.34)
  - Updating symfony/form (v2.8.28 => v2.8.34)
  - Updating symfony/validator (v2.8.28 => v2.8.34)
  - Updating symfony/polyfill-apcu (v1.6.0 => v1.7.0)
  - Updating symfony/cache (v3.3.10 => v3.4.4)
  - Updating symfony/class-loader (v2.8.28 => v2.8.34)
  - Updating symfony/polyfill-util (v1.6.0 => v1.7.0)
  - Updating symfony/polyfill-php70 (v1.6.0 => v1.7.0)
  - Updating symfony/polyfill-php56 (v1.6.0 => v1.7.0)
  - Updating symfony/polyfill-php55 (v1.6.0 => v1.7.0)
  - Updating symfony/security (v2.8.28 => v2.8.34)
  - Updating symfony/http-foundation (v2.8.28 => v2.8.34)
  - Updating symfony/http-kernel (v2.8.28 => v2.8.34)
  - Updating symfony/security-bundle (v2.8.28 => v2.8.34)
  - Updating symfony/monolog-bridge (v2.8.28 => v2.8.34)
  - Updating symfony/asset (v2.8.28 => v2.8.34)
  - Updating symfony/twig-bundle (v2.8.28 => v2.8.34)
  - Updating symfony/console (v2.8.28 => v2.8.34)
  - Updating doctrine/orm (v2.5.12 => v2.5.14)
  - Updating symfony/doctrine-bridge (v2.8.28 => v2.8.34)
  - Updating symfony/dependency-injection (v2.8.28 => v2.8.34)
  - Updating symfony/templating (v2.8.28 => v2.8.34)
  - Updating symfony/stopwatch (v2.8.28 => v2.8.34)
  - Updating symfony/routing (v2.8.28 => v2.8.34)
  - Updating symfony/finder (v2.8.28 => v2.8.34)
  - Updating symfony/filesystem (v2.8.28 => v2.8.34)
  - Updating symfony/config (v2.8.28 => v2.8.34)
  - Updating symfony/framework-bundle (v2.8.28 => v2.8.34)
  - Updating doctrine/doctrine-bundle (1.6.13 => 1.8.1)
  - Updating knplabs/knp-menu (2.2.0 => 2.3.0)
  - Updating knplabs/knp-menu-bundle (2.1.3 => v2.2.1)
  - Updating oneup/uploader-bundle (1.8.3 => 1.9.2)
  - Updating composer/ca-bundle (1.0.8 => 1.1.0)
  - Updating maxmind/web-service-common (v0.4.0 => v0.5.0)
  - Updating geoip2/geoip2 (v2.7.0 => v2.8.0)
  - Updating giggsey/locale (1.3 => 1.4)
  - Updating giggsey/libphonenumber-for-php (8.8.4 => 8.8.11)
  - Updating misd/phone-number-bundle (v1.2.0 => v1.3.2)
  - Updating twilio/sdk (4.12.0 => 4.12.1)
  - Updating stack/builder (v1.0.4 => v1.0.5)
  - Updating sonata-project/exporter (1.7.1 => 1.8.0)
  - Updating symfony/expression-language (v2.8.28 => v2.8.34)
  - Updating symfony/dotenv (v3.3.10 => v3.4.4)
  - Updating symfony/yaml (v2.8.28 => v2.8.34)
  - Updating php-amqplib/php-amqplib (v2.7.0 => v2.7.2)
  - Updating php-amqplib/rabbitmq-bundle (v1.13.0 => v1.14.2)
  - Updating leezy/pheanstalk-bundle (3.2.1 => 3.2.3)
  - Updating simshaun/recurr (v3.0.2 => v3.0.5)
  - Updating ramsey/uuid (3.7.1 => 3.7.3)
  - Updating symfony/web-profiler-bundle (v2.8.28 => v2.8.34)
  - Updating symfony/dom-crawler (v2.8.28 => v2.8.34)
  - Updating symfony/browser-kit (v2.8.28 => v2.8.34)
  - Updating liip/functional-test-bundle (1.8.0 => 1.9.3)
  - Updating symfony/process (v2.8.28 => v2.8.34)
  - Updating sensio/generator-bundle (v3.1.6 => v3.1.7)
  - Updating symfony/polyfill-php72 (v1.6.0 => v1.7.0)
  - Updating gecko-packages/gecko-php-unit (v2.2 => v2.2.1)
  - Updating friendsofphp/php-cs-fixer (v2.2.8 => v2.2.16)
  - Updating symfony/var-dumper (v3.3.10 => v3.4.4)
  - Updating symfony/css-selector (v3.3.10 => v3.4.4)
  - Updating phpunit/php-file-iterator (1.4.2 => 1.4.5)
  - Updating phpunit/php-token-stream (1.4.11 => 1.4.12)
  - Updating webmozart/assert (1.2.0 => 1.3.0)
  - Updating phpdocumentor/type-resolver (0.3.0 => 0.4.0)
  - Updating phpdocumentor/reflection-docblock (3.2.2 => 3.3.2)
  - Updating phpspec/prophecy (v1.7.2 => 1.7.4)
  - Updating phpunit/phpunit (5.7.23 => 5.7.27)
  - Updating facebook/webdriver (1.4.1 => 1.5.0)
  - Installing codeception/stub (1.0.1)
  - Updating codeception/codeception (2.3.6 => 2.3.8)
  - Updating friendsofsymfony/oauth2-php (1.2.1 => 1.2.2)
  - Updating php-http/discovery (1.3.0 => 1.4.0)
  - Updating swiftmailer/swiftmailer (v5.4.8 => v5.4.9)
  - Updating sensiolabs/security-checker (v4.1.6 => v4.1.7)
  - Updating jms/serializer (1.9.1 => 1.11.0)
  - Updating lightsaml/lightsaml (1.3.1 => 1.3.3)
```

### Minor version upgrades:

I went through all changes logs of packages that had a major version update:

- https://github.com/symfony/polyfill-mbstring - no change log
- https://github.com/symfony/polyfill-php54 - no change log
- https://github.com/symfony/polyfill-intl-icu - no change log
- https://github.com/symfony/polyfill-apcu - no change log
- https://github.com/symfony/polyfill-util - no change log
- https://github.com/symfony/polyfill-php70 - no change log
- https://github.com/symfony/polyfill-php56 - no change log
- https://github.com/symfony/polyfill-php55 - no change log
- https://github.com/symfony/polyfill-php72 - no change log
- https://github.com/doctrine/DoctrineBundle/releases, but https://github.com/doctrine/DoctrineBundle/issues/780
- https://github.com/KnpLabs/KnpMenu/releases/tag/2.3.0
- https://github.com/KnpLabs/KnpMenuBundle/blob/master/CHANGELOG.md
- https://github.com/1up-lab/OneupUploaderBundle#upgrade-notes
- https://github.com/composer/ca-bundle/releases
- https://github.com/maxmind/web-service-common-php/releases
    - Refer to account IDs using the terminology "account" rather than "user". - can be problem for us?
- https://github.com/maxmind/GeoIP2-php/blob/master/CHANGELOG.md#280-2018-01-18
- https://github.com/giggsey/Locale/releases/tag/1.3
- https://github.com/misd-service-development/phone-number-bundle/blob/master/CHANGELOG.md#130
- https://github.com/sonata-project/exporter/blob/1.x/CHANGELOG.md
    - Support for old versions of PHP and Symfony. means `"php": "^5.6 || ^7.0"`
- https://github.com/symfony/dotenv/blob/master/CHANGELOG.md but 3.4 is not documented
- https://github.com/php-amqplib/RabbitMqBundle/releases/tag/v1.14.0
- https://github.com/liip/LiipFunctionalTestBundle/releases/tag/1.9.
- https://github.com/symfony/var-dumper/blob/master/CHANGELOG.md#340
    - move to dev dependencies **???**
- https://github.com/symfony/css-selector/blob/master/CHANGELOG.md
    - outdated, does not have 3.4.0
- https://github.com/webmozart/assert/blob/master/CHANGELOG.md
- https://github.com/phpDocumentor/TypeResolver/releases/tag/0.4.
- https://github.com/phpDocumentor/ReflectionDocBlock/releases/tag/3.3.0
- https://github.com/facebook/php-webdriver/blob/community/CHANGELOG.md#150---2017-11-15
- https://github.com/php-http/discovery/blob/master/CHANGELOG.md#140---2018-02-06
- https://github.com/schmittjoh/serializer/blob/master/CHANGELOG.md

#### Steps to test this PR:
1. Checkout this PR
2. Run `composer install`
3. Clearing cache with a PHP command may not be enough. `rm -rf app/cache/*` to make sure cache won't misbehave.
3. Everything should work. Regression testing of all features is needed.

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -645,41 +642,38 @@
         },
         {
             "name": "doctrine/doctrine-bundle",
-            "version": "1.6.13",
+            "version": "1.6.0",
```
Comment (Maxell92) [inline composer.lock]: Is this really last possible version? We have downgraded here.
I can see changes that can prevent us to include last 1.6.x version, but do we really need a 1.6.0?
https://github.com/doctrine/DoctrineBundle/compare/1.6.0...1.6.13

Comment (escopecz) [reply composer.lock]: That was Composer's decision based on [telling him](https://github.com/mautic/mautic/pull/5718/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780R153) Mautic's PHP version requirement. We can try to force some newer version. Just have to figure out which one.

**Comment by Maxell92** [comment]
`composer install` showed an error:
```Warning: The lock file is not up to date with the latest changes in composer.json. You may be getting outdated dependencies. Run update to update them.```

It was caused by difference of `doctrine/doctrine-bundle` in composer.json` and `composer.lock`. I update this library back to 1.6.13 version, which is the latest one we can use.

Tested and it seems everything is working.

**Comment by Dreiser** [comment]
Works for me, hope it will work all in .13

**Pull request merged**